spec/helpers/searches_helper_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+#DO NOT DELETE - this is needed to run RSPEC for the search logic
```

Comment:
I imagine there's a way around the strange behaviour that we observed, but fixing it doesn't need to be in this PR.